crates/moon/tests/test_cases/specify_source_dir_001.in/moon.test

```diff
@@ -68,6 +69,7 @@
     "source": "src"
   }
   $ moon build
+  
```

Comment:
why add the newline 😯

Reply:
See xtask/src/cmdtest/exec.rs:76 : If stdout is empty and stderr, the result will be `"\n{stderr}"`.

```rust
impl ExecResult {
    pub fn normalize(&self, workdir: &Path) -> String {
        let actual = if self.stderr.is_empty() {
            self.stdout.clone()
        } else {
            format!("{}\n{}", self.stdout, self.stderr) // <-
        };
        let actual = if self.exit_code != 0 {
            format!("[{}]\n{}", self.exit_code, actual)
        } else {
            actual
        };
        let actual = replace_dir(&actual, &workdir);
        actual
    }
}
```